### DISCUSSION TITLE
How to disable new Turborepo Terminal UI in CI but keep it enabled for local development?

### DISCUSSION THREAD

**Discussion by yashsway** [vercel/turborepo#8900] [Help] [answered]
### Summary

So I've got the new terminal UI working with https://turbo.build/repo/docs/reference/configuration#ui but this basically sets the terminal UI for every pipeline.... and some pipelines are only run in CI so I don't want the terminal UI there. So I saw this setting https://turbo.build/repo/docs/reference/run#--log-order-option but it turns out this doesn't work with the new terminal UI setting. 

So how does one go about disabling this UI for some pipelines? or in a CI environment for example?

**Comment by anthonyshew** [comment, marked as answer]
From [your other post](https://github.com/vercel/turbo/discussions/8905), it looks like you're using CircleCI, which improperly sets TTY. Per [our CircleCI doc](https://turbo.build/repo/docs/guides/ci-vendors/circleci), you'll need to set `TURBO_UI=false` to get back to the log streaming UI.
